src/typing/flow_error.ml

```diff
@@ -1413,3 +1415,9 @@ let rec error_of_msg ~trace_reasons ~op ~source_file =
         null_loc, ["Potentially null/undefined value."];
         falsy_loc, [spf "%s value." value_str]
       ]]
+  | EUnknownProperty (loc, name) ->
+    mk_error
+      ~kind:(LintError LintSettings.UnknownProperty)
```

Comment:
Try temporarily making this an error and running our test suite. I’d imagine that this error would show up in some unexpected places.